src/components/SocialMediaLinks/SocialMediaLinks.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import SocialMediaLinks, {
+  borderTypeArray,
+  colorTypeArray,
+  sizeTypeArray,
+} from "./SocialMediaLinks";
+import { withDesign } from "storybook-addon-designs";
+import { layoutTypesArray } from "../../helpers/types";
+
+const meta: Meta<typeof SocialMediaLinks> = {
+  title: "Components/Navigation/SocialMediaLinks",
+  component: SocialMediaLinks,
+  decorators: [withDesign],
+  argTypes: {
+    borders: {
+      control: { type: "radio" },
+      options: borderTypeArray,
+      table: { defaultValue: { summary: "none" } },
+    },
+    color: {
+      control: { type: "radio" },
+      options: colorTypeArray,
+      table: { defaultValue: { summary: "textDefault" } },
+    },
+    layout: {
+      control: { type: "radio" },
+      options: layoutTypesArray,
+      table: { defaultValue: { summary: "row" } },
+    },
+    showLabels: { table: { defaultValue: { summary: false } } },
+    size: {
+      control: { type: "radio" },
+      options: sizeTypeArray,
+      table: { defaultValue: { summary: "small" } },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof SocialMediaLinks>;
+
+/*
+ *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+ * See https://storybook.js.org/docs/react/api/csf
+ * to learn how to use render functions.
+ */
+export const Primary: Story = {
```

Comment:
Check out the `Button` example: https://github.com/NYPL/nypl-design-system/blob/development/src/components/Button/Button.stories.tsx#L66

Instead of `Primary`, we call the main exported story `WithControls`. This variables defines the `args` and `parameters` properties and in the render function, the `args` must be passed to the component. In this can you might be able to do `render: (args) => <SocialMediaLinks {...args} />`.

Doing this will make the Storybook "Controls" tab work with the component, so that when a value is updated in the UI, it gets propagated down to the component.